CHANGELOG.rst

```diff
@@ -1,5 +1,23 @@
 Changelog
 =========
+1.19.0 (2021-07-14)
+-------------------
+MLflow 1.19.0 includes several major features and improvements:
+
+Features:
+
+- Add support for plotting per-class feature importance computed on linear boosters in XGBoost autologging (#4523, @dbczumar)
+- Add ``mlflow_create_registered_model`` and ``mlflow_delete_registered_model`` for R to create/delete registered models.
+- Add support for setting tags while resuming run (#4497, @dbczumar)
+- Syncing with MLFlow and adding i18n support (#4490, @sunishsheth2009)
```

Comment:
https://github.com/mlflow/mlflow/pull/4490

Reply:
- Adding i18n support to OSS
- Moving metrics to show before parameters and tags in the runs table
- `startTime` format change (timestamp -> elapsed time)

Reply:
Sounds good! Let's call those out!